src/object.ts

```diff
@@ -1,23 +1,40 @@
 import { genObjectKey, wrapInDelimiters } from "./utils";
 
-export function genObjectFromRaw (object: Record<string, any>, indent = ""): string {
+export function genObjectFromRaw(
+  object: Record<string, any>,
+  indent = ""
+): string {
   return genObjectFromRawEntries(Object.entries(object), indent);
 }
 
-export function genArrayFromRaw (array: any[], indent = "") {
+export function genArrayFromRaw(array: any[], indent = "") {
   const newIdent = indent + "  ";
-  return wrapInDelimiters(array.map(index => `${newIdent}${genRawValue(index, newIdent)}`), indent, "[]");
+  return wrapInDelimiters(
+    array.map((index) => `${newIdent}${genRawValue(index, newIdent)}`),
+    indent,
+    "[]"
+  );
 }
 
-export function genObjectFromRawEntries (array: [key: string, value: any][], indent = "") {
+export function genObjectFromRawEntries(
+  array: [key: string, value: any][],
+  indent = ""
+) {
   const newIdent = indent + "  ";
-  return wrapInDelimiters(array.map(([key, value]) => `${newIdent}${genObjectKey(key)}: ${genRawValue(value, newIdent)}`), indent, "{}");
+  return wrapInDelimiters(
+    array.map(
+      ([key, value]) =>
+        `${newIdent}${genObjectKey(key)}: ${genRawValue(value, newIdent)}`
+    ),
+    indent,
+    "{}"
+  );
 }
 
 // --- Internals ---
 
-function genRawValue (value: unknown, indent = ""): string {
-  if (typeof value === "undefined") {
+function genRawValue(value: unknown, indent = ""): string {
+  if (typeof value === "undefined" || value === undefined) {
```

Comment:
I added `value === undefined` condition, othewise `value` can't be type-narrowed at the point of `value.toString()`.

Reply:
@pi0 

I don't know why, but TypeScript seems to not type-narrow value with `typeof x === "undefined"` condition.

Since it passes unit tests, let me merge this PR for now.
But, please be aware that I included code change in this PR as well as applying the latest template. 🙌